plaidml/bridge/openvino/tests/functional/shared_tests_instances/single_layer_tests/strided_slice.cpp

```diff
@@ -7,181 +7,290 @@
+        {1, 0, 1},
+        {1, 0, 1},
+        {0, 0, 0},
+        {0, 0, 0},
+        {0, 0, 0},
+    },
+    StridedSliceSpecificParams{
+        {1, 12, 100},
+        {0, 10, 0},
+        {0, 4, 0},
+        {-1, -2, -1},
+        {1, 0, 1},
+        {1, 0, 1},
+        {0, 0, 0},
+        {0, 0, 0},
+        {0, 0, 0},
+    },
+    StridedSliceSpecificParams{
+        {1, 12, 100},
+        {0, 11, 0},
+        {0, 0, 0},
+        {-1, -2, -1},
+        {1, 0, 1},
+        {1, 0, 1},
+        {0, 0, 0},
+        {0, 0, 0},
+        {0, 0, 0},
+    },
+    StridedSliceSpecificParams{
+        {1, 12, 100},
+        {0, -6, 0},
+        {0, -8, 0},
+        {-1, -2, -1},
+        {1, 0, 1},
+        {1, 0, 1},
+        {0, 0, 0},
+        {0, 0, 0},
+        {0, 0, 0},
+    },
+    StridedSliceSpecificParams{
+        {1, 12, 100, 1, 1},
+        {0, -1, 0, 0},
+        {0, 0, 0, 0},
+        {1, 1, 1, 1},
+        {1, 0, 1, 0},
+        {1, 0, 1, 0},
+        {},
+        {0, 1, 0, 1},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 2, 2},
+        {0, 0, 0, 0},
+        {2, 2, 2, 2},
+        {1, 1, 1, 1},
+        {1, 1, 1, 1},
+        {1, 1, 1, 1},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 2, 2},
+        {1, 1, 1, 1},
+        {2, 2, 2, 2},
+        {1, 1, 1, 1},
+        {0, 0, 0, 0},
+        {1, 1, 1, 1},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 2, 2},
+        {1, 1, 1, 1},
+        {2, 2, 2, 2},
+        {1, 1, 1, 1},
+        {0, 0, 0, 0},
+        {0, 0, 0, 0},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 4, 3},
+        {0, 0, 0, 0},
+        {2, 2, 4, 3},
+        {1, 1, 2, 1},
+        {1, 1, 1, 1},
+        {1, 1, 1, 1},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 4, 2},
+        {1, 0, 0, 1},
+        {2, 2, 4, 2},
+        {1, 1, 2, 1},
+        {0, 1, 1, 0},
+        {1, 1, 0, 0},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {1, 2, 4, 2},
+        {1, 0, 0, 0},
+        {1, 2, 4, 2},
+        {1, 1, -2, -1},
+        {1, 1, 1, 1},
+        {1, 1, 1, 1},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 2, 4, 2},
+        {1, 0, 0, 0},
+        {1, 2, 4, 2},
+        {1, 1, -2, -1},
+        {0, 1, 1, 1},
+        {1, 1, 1, 1},
+        {},
+        {},
+        {},
+    },
+    StridedSliceSpecificParams{
+        {2, 3, 4, 5, 6},
+        {0, 1, 0, 0, 0},
+        {2, 3, 4, 5, 6},
+        {1, 1, 1, 1, 1},
+        {1, 0, 1, 1, 1},
+        {1, 0, 1, 1, 1},
+        {},
+        {0, 1, 0, 0, 0},
+        {},
+    },
+
+    /// those two case will run into segment fault, it is caused by openvino inference,
+    /// which is locate in 'MakeNextStageTask' of 'ie_infer_async_request_thread_safe_default.hpp'
```

Comment:
Can we have a longer discussion on the problem here? A key question: does this also cause problems for the CPU plugin?

Reply:
no, this don't cause problems for the CPU plugin, we are still further check this issue, and will fix this in a following PR.